spec/enum_message_spec.rb

```diff
@@ -0,0 +1,113 @@
+RSpec.describe "enum message encoding" do
+  # enum Crunchiness {
+  #   FATTY  = 0;
+  #   CRISPY = 1;
+  #   CEMENT = 2;
+  # }
+  # message Bacon {
+  #   required uint32 id = 1;
+  #   optional Crunchiness chrunchiness = 2;
+  # }
+  class Crunchiness
+    VALUES_BY_LABEL = {
```

Comment:
I would think this would include a path that would get to the enum value directly as well like

so the Enum type is still usable like an enum (but this may be my C/C++ bias showing)

```ruby
class Crunchiness
  FATTY = Crunchiness.new(:FATTY)
  CRISPY = Crunchiness.new(:CRISPY)
  CEMENT = Crunchiness.new(:CEMENT)
end
```